package.json

```diff
@@ -19,10 +20,12 @@
     "dev:pack": "turbo dev:pack --concurrency 16",
     "format": "turbo format",
     "format:prettier": "prettier --write .",
+    "format:pretty-quick": "pretty-quick",
     "format:syncpack": "syncpack format",
-    "lint": "turbo lint -- --max-warnings 0",
```

Comment:
per your suggestion, should we apply `--max-warnings 0` to the ci workflows?

Reply:
yes that's preferable, i'll make ci use a per-package `lint:strict` so that

1. acceptable warnings can be configured per-package (nice for new rule introductions)
2. a typical manual call to `pnpm lint` can still show warnings to a developer
3. `lint:strict` can include additional checks like adding a `tsc --noEmit` call